alcs-frontend/src/app/features/admin/board-management/board-management-dialog/board-management-dialog.component.ts

```diff
@@ -98,7 +100,7 @@ export class BoardManagementDialogComponent implements OnInit {
     this.step++;
     //We don't support creating Notifications
     this.allowedCreateCardTypes = (this.permittedCardTypes.value ?? []).filter(
-      (cardType) => cardType !== CardType.NOTIFICATION
+      (cardType) => !DISABLED_CREATE_CARD_TYPES.includes(cardType),
```

Comment:
Won't this now enable `NOTIFICATION` card type? Is that intended?

Reply:
It definitely would and not intended. Good catch! MR updated.